loc/lcl/PLK/BoostTestAdapter.dll.lcl

```diff
@@ -176,7 +176,7 @@
         <Str Cat="Text">
           <Val><![CDATA[Finding processes spawned by process with Id [{0}]5D;]]></Val>
           <Tgt Cat="Text" Stat="Loc" Orig="New">
-            <Val><![CDATA[Znajdowanie procesów utworzonych przez proces o identyfikatorze [{0}]5D;]]></Val>
+            <Val><![CDATA[Znajdowanie procesów rozmnożonych przez proces o identyfikatorze [{0}]5D;]]></Val>
```

Comment:
Oh boy, this needs to be reverted. It switched the translation from "spawn (computing)" to "spawn (biology)" 😐.